spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
Maybe. I can give it a try. It's certainly not obvious when reading the spec that there's an implicit "current execution context" -> implicitly created PendingJob's [[Realm]] field -> continent link.